### ISSUE TITLE
New TRAN EDW issue

### ISSUE THREAD

**Issue by jon-mc-git** [bcgov/cloud-pathfinder#1663]
**Describe the issue**
New issue with TRAN involving data transfer.  Comms issue is between the Integrated Runtime (IR) servers, which live in 2 redundant VMs in their IP space and the data factory.

**Additional context**
- Jon has had a meeting with Microsoft, but it the issue is still not getting clearer

**Definition of done**
- Help Tran EDW and Microsoft
- Monkey wrenches get separate tickets

**Comment by jon-mc-git** [comment]
Had to explore additional more specialized Microsoft resources because the current MS team assign to TRAN could not figure it out.  Azure ticket was created to get specialized resource in Azure DataFactory and the use of IR VM in a private link environment (to Internet access).  Met with the new resource here and they indicated that the limited access that is being mentioned in the gui and logs is ok for basic usage, so the TRAN technical resource could go ahead still with its use.  However, certain functionality will NOT be available, as these specific functions do require the use of public (Internet) based URL's.  Currently trying to get the IP list from MS around what these URL's translate to so we could try a bypass, however URL's normally require a domain-based Forward Proxy service (where domain\url's can be whitelisted - as IP's can change over time and this takes care of that).  We currently to not have a Proxy service in place for cloud that allows for this.  A new ticket was created to explore getting this capability.  In meantime waiting on IP list from MS to see if a bypass might be possible (where Azure grab it and route internally to their local resources)

**Comment by jon-mc-git** [comment]
Have gone back and forth with Microsoft and proposed a solution which did not work.  They have come back and proposed something else but from what I can see they need to get service\network comm tracer involved.  They have made the request but resources are tight their so putting this on hold as blocked until I get further work from them

**Comment by ActionAnalytics** [comment]
Let's close this ticket now, and what we'll do is update the CRM with this ticket link, and say that we expect another round later, and when we do, we will check CRM, see this (closed) ticket and link the new one we'll make at that point to this one.  That way we clear the board without losing any of the nice context we have here.
